docs/installing.md

```diff
@@ -60,9 +60,9 @@ Once that message shows up, open a web browser and connect to [http://localhost:
 
 When you are finished with PrairieLearn, type Control-C on the terminal where you ran the server to stop it.
 
-### Running on ARM64 hardware
+### Running on Apple M1 and other ARM64 hardware
```

Comment:
```suggestion
### Running on Apple Silicon and other ARM64 hardware
```

Reply:
I can make this change, but I tried to refer to "M1" in the body text since not everybody knows that Apple calls their chip line "Apple Silicon". Apple themselves don't capitalize the word "silicon" and they didn't try to trademark it, from what I can tell, so it's sort of vague whether they actually mean this to be the product line's name, or they just expect people to understand that silicon means "chips".

Reply:
You can lowercase silicon, or refer to "Apple M-series processors", but I don't want to hardcode references to specific versions (M1 or otherwise).

Reply:
I made it lowercase and changed "M1, M2, etc." to "M-series chips" in the body text. Coincidentally Apple also insists on referring to the Intel processors as "processors" and Apple silicon as "chips," not "processors." I don't mind any of this but it's amusing to note.